src/coreclr/vm/jitinterface.cpp

```diff
@@ -10755,25 +10751,28 @@ void CEECodeGenInfo::getHelperFtn(CorInfoHelpFunc    ftnNum,               /* IN
                                    CORINFO_CONST_LOOKUP* pNativeEntrypoint, /* OUT */
                                    CORINFO_METHOD_HANDLE* pMethod)          /* OUT */
 {
-    CONTRACTL {
+    CONTRACTL
+    {
         THROWS;
         GC_TRIGGERS;
         MODE_PREEMPTIVE;
-    } CONTRACTL_END;
+    }
+    CONTRACTL_END;
 
     JIT_TO_EE_TRANSITION();
 
-    if (pMethod != NULL)
-    {
-        *pMethod = NULL;
-    }
-
     _ASSERTE(ftnNum < CORINFO_HELP_COUNT);
 
-    void* pfnHelper = hlpFuncTable[ftnNum].pfnHelper;
+    InfoAccessType accessType = IAT_PVALUE;
+    LPVOID targetAddr = nullptr;
 
-    size_t dynamicFtnNum = ((size_t)pfnHelper - 1);
-    if (dynamicFtnNum < DYNAMIC_CORINFO_HELP_COUNT)
+    MethodDesc* helperMD = NULL;
+    VMHELPDEF const& helperDef = hlpFuncTable[ftnNum];
+    TADDR pfnHelper = helperDef.pfnHelper;
+
+    size_t dynamicFtnNum;
+    bool isDynamicMethod = helperDef.IsDynamicHelper(dynamicFtnNum);
+    if (isDynamicMethod)
```

Comment:
```suggestion
    bool isDynamicHelper = helperDef.IsDynamicHelper(dynamicFtnNum);
    if (isDynamicHelper)
```
DynamicMethod is a thing that has nothing to do with this.

Reply:
Oops. I thought I replaced all those.